README.md

```diff
@@ -33,9 +33,11 @@ This is a community effort in building a pipeline capable to support:
 
 The nf-core/scrnaseq pipeline comes with documentation about the pipeline [usage](https://nf-co.re/scrnaseq/usage), [parameters](https://nf-co.re/scrnaseq/parameters) and [output](https://nf-co.re/scrnaseq/output).
 
+![scrnaseq workflow](docs/images/scrnaseq_pipeline_v1.0_metro_clean.png)
```

Comment:
should add universc in the pipeline chart

Reply:
I think the chart was created by @heylf... Would you have time to update it?

Reply:
Yes, I can put it in.

Reply:
that would be amazing!

Reply:
@grst https://github.com/nf-core/scrnaseq/pull/209